### PR TITLE
Update F-DDrace entities by Souly

### PR DESCRIPTION
**Pull request by fokkonaut**
@SoulyVEVO 

## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: data/editor/entities/F-DDrace.png
Comment (def-) [inline data/editor/entities/F-DDrace.png:1]: Is this being used even though it has F-DDrace as a name instead of f-ddrace?

Comment (fokkonaut) [reply data/editor/entities/F-DDrace.png:1]: Yes, client check case insensitive.

Comment (fokkonaut) [reply data/editor/entities/F-DDrace.png:1]: Btw: this is for the editor. It uses the name from the file to display it

**Review by def- — APPROVED** [review]

**Comment by heinrich5991** [comment]
@def- Especially for images, please squash commits so that we don't include large files in the repository twice.

**Pull request merged**